### DISCUSSION TITLE
Swissup Ajax pro with Breeze

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#287] [1. Help]
Hi @vovayatsyuk 

Sometimes Swissup ajax pro added to cart modal opens on page load which should be only when we add any product, with breeze and this is happening in all our projects where used both these together.

**Comment by vovayatsyuk** [comment]
I've noticed that happens with Luma too. Do you have some scenarios on how to reproduce it?

**Reply by pk-pankaj** [reply]
@vovayatsyuk I am trying to catch the scenarios for few days but could not do as it's not making any scenarios it comes randomly. 
It has been serious issue for one of our client so really need help on this if you can.

**Reply by pk-pankaj** [reply]
We also created swissup support ticket for this issue, fixed in new release 1.7.9 as they are saying so we will check by updating the module.

**Reply by pk-pankaj** [reply]
@vovayatsyuk  this also happening with the latest versions.
